user/src/main/java/org/dicadeveloper/weplantaforest/projects/Price.java

```diff
@@ -37,6 +41,7 @@
     private ScontoType scontoType = ScontoType.NONE;
 
     @Column(name = "_amount", precision = 7, scale = 2)
+    @JsonView({Views.Project.class})

```

Comment:
her's an example for the first object when calling /projects:
sorry for the bad formatting, it seems, that whitespaces won't be recognized here

 "0": {
    "name": "Project 0 von Johannes",
    "description": "dksgny.d, mdfgnmn snfad,ng ,ydfng. ,ydfgnk.<sngdk< sglkbsnglkdfnksghnl<k njdjg nsgyö< ögn kl< bsflkjsb gkjs kgs ns< lödgksndlkgnöd<kl dykdyn ökd ökshö<g dysh ögskgös Hskg khoglksg",
    "longitude": 13.607287,
    "latitude": 52.462654,
    "manager": {
      "name": "Johannes"
    },
    "articles": [
      {
        "articleId": 1,
        "treeType": {
          "name": "Buche",
          "annualCo2SavingInTons": 0.02
        },
        "price": {
          "amount": 2.26
        },
        "amount": 9
      },
      {
        "articleId": 2,
        "treeType": {
          "name": "Kiefer",
          "annualCo2SavingInTons": 0.02
        },
        "price": {
          "amount": 5.32
        },
        "amount": 483
      },
      {
        "articleId": 3,
        "treeType": {
          "name": "Birke",
          "annualCo2SavingInTons": 0.02
        },
        "price": {
          "amount": 1.35
        },
        "amount": 412
      }
   ]
  }